rotkehlchen/tests/api/test_user_evm_tokens.py

```diff
@@ -577,3 +577,30 @@ def test_user_tokens_delete_guard(rotkehlchen_api_server):
         contained_in_msg=expected_msg,
         status_code=HTTPStatus.CONFLICT,
     )
+
+
+def test_non_ethereum_token(rotkehlchen_api_server):
+    response = requests.put(
```

Comment:
```suggestion
def test_adding_non_ethereum_token(rotkehlchen_api_server):
    """Test that adding evm tokens from non ethereum chains works as expected"""
    response = requests.put(
```

Reply:
Isn't it exactly the same as the function's name says?

Reply:
yes and no but at least change the test name. It has happened to me that sometimes there is no description and the name is bad and is hard to figure out what the test wanted to check